data/css/layout.css

```diff
@@ -26,4 +26,34 @@
 
 .border-bottom {
     border-bottom: 1px solid shade (@bg_color, 0.8);
-}
\ No newline at end of file
+}
+
+.entry-label {
+  text-shadow: 0 1px @text_shadow_color;
+  background-image:
+  linear-gradient(
+        to top,
+        shade (
+            @base_color,
+            0.93
+        ),
+        shade (
+            @base_color,
+            0.97
+        )
+    );
+  border: 1px solid @border_color;
+  box-shadow:
+      inset 0 0 0 1px alpha (@bg_highlight_color, 0.05),
+      inset 0 -1px 0 0 alpha (@bg_highlight_color, 0.45),
+      inset 0 1px 0 0 alpha (@bg_highlight_color, 0.15),
+      0 1px 0 0 alpha (@bg_highlight_color, 0.15);
+}
+
+.linked .entry-label:first-child {
```

Comment:
I'm not sure if this `.linked` styling could affect other widgets as that class already exists and it's used throughout the Gtk stylesheet.

Reply:
It only targets `.entry-label` inside `.linked`, not `.linked` directly, so I think there is no problem with that.

Reply:
It's pretty baffling that GTK doesn't provide a linked style for entry labels.
This could be a great upstream PR 😃️